Projects/IntelligentObjectAvoider/README.md

```diff
@@ -8,7 +10,7 @@ In this project, we make the `GoPiGo3` intelligently avoid obstacles. You need t
 
 The `servo` is particularly useful, because it gives the distance sensor a 120 degrees viewing angle. The script uses a greedy-like algorithm for choosing the best path.
 
-In order to test this script, it's advised to create at your place a small labyrinth where the `GoPiGo3` can be put in. Books, bags, stools can be used. Avoid creating very sharp corners in your little maze or too small "corridors" as your `GoPiGo3` can get stuck.
+## Preparing It
 
 You need to do the following things in order to be able to launch the `robot.py` script:
 1. Install `scikit-learn` package. A `pip install scikit-learn` should be enough (be mindful of whether you need to use `sudo` or not - most probably it's `sudo` that's required).
```

Comment:
wouldn't that be `pip3 install scikit-learn`  ? 
I lost a good chunk of time yesterday as I was following the instructions blindly and ended up installing scikit-learn twice.

Reply:
This matter is super subjective. If you're using a virtualenv or conda it can be `pip`, if you do it from the system wide space, it could be `pip3` as well as `pip`.

Reply:
In cases like this, you cater to the beginner who's most likely on Raspbian for Robots and unable to fall back on their feet. Anyone dealing with conda or virtualenv knows how to work around it.  Not the beginner.